web/src/features/hfiCalculator/components/CalculatedPlanningAreaCells.tsx

```diff
@@ -66,26 +46,23 @@ const CalculatedPlanningAreaCells = (props: CalculatedCellsProps) => {
               dailies={dailies ? dailies : []}
               selectedStations={props.selected}
               meanIntensityGroup={meanIntensityGroup}
-            ></MeanIntensityGroupRollup>
-            <FireStartsCell areaName={props.areaName} />
-            <PrepLevelCell
-              meanIntensityGroup={meanIntensityGroup}
-              areaName={props.areaName}
             />
+            <FireStartsCell areaName={props.areaName} />
+            <PrepLevelCell prepLevel={prepLevel} />
           </React.Fragment>
         )
       })}
 
       <MeanIntensityGroupRollup
         area={props.area}
-        dailies={areaDailies}
+        dailies={props.areaHFIResults.dailies}
         selectedStations={props.selected}
-        meanIntensityGroup={highestMeanIntensityGroup}
+        meanIntensityGroup={props.areaHFIResults.maxMeanIntensityGroup}
       ></MeanIntensityGroupRollup>
       <AveragePrepLevelCell
-        meanIntensityGroup={meanPrepLevel}
+        meanIntensityGroup={props.areaHFIResults.meanPrepLevel}
         areaName={props.areaName}
-        meanPrepLevel={meanPrepLevel}
+        meanPrepLevel={props.areaHFIResults.meanPrepLevel}
```

Comment:
I know that all you've done in this change is renaming a variable, but just wanted to flag to make sure this isn't a bug - why are we passing `meanPrepLevel` to the AveragePrepLevelCell component as both `meanIntensityGroup` and `meanPrepLevel`?

Reply:
Thanks, will take another look. I appreciate you looking at these details they are definitely a vector for bugs when names are similar.

Reply:
`meanIntensityGroup` is not even used anymore! I forgot to delete it in the props. Also I'll rename `Average` to `Mean` to be more explicit.

Reply:
fabulous, thanks for looking into that Conor!